src/App.tsx

```diff
@@ -31,23 +19,34 @@ function App() {
     setIsNewCardDisplayed(true);
   };
 
-  const handleSaveButtonClick = (card: CardObject) => {
-    setCardsArray([
-      { face: card.face, back: card.back },
-      ...cardsArray,
-    ]);    
+  const handleSaveButtonClick = async (card: CardObject) => {
+    await exportCard(card);
```

Comment:
**nitpick:** `Export`, is the same like `import` - it's registered keyword for doing very specific thing. We should not name `post` functions with it. Let's change it to `postNewCard` or something similar